### DISCUSSION TITLE
Can I Set a variable equal to a scene item's transform value or filter property value?

### DISCUSSION THREAD

**Discussion by russellwyner** [WarmUpTill/SceneSwitcher#685] [Q&A]
I love that you added variables. Being able to put ${Variable Name} into any property's text field is amazing. 

Now, I'm trying to set a variable equal to the x-position value of a scene item, or really any value of a transform or filter property. Is there a way to do this? Is there a syntax for typing out a scene item's specific transform or filter property in the text field for "Set to fixed value"? When I use "Set to condition value" it writes out the full array of defined values. I can't target a specific transform property or filter value.

**Comment by WarmUpTill** [comment]
You will have to first set the value of a variable to the full array of values and then pick out specific one by using the option to "Set to substring of current value".
You will have to provide a pattern that matches particular line / value you want assign to the variable.

I don't have access to OBS at the moment so I can only reference an existing example showcasing this functionality:
https://github.com/WarmUpTill/SceneSwitcher/issues/664

**Reply by russellwyner** [reply]
It worked! I hadn't thought of separating it into two parts. And then  I just used `\d+ ` to get the number values. Thank you so much for that quick response!
